### DISCUSSION TITLE
My Old github account has been hacked

### DISCUSSION THREAD

**Discussion by ghost** [community/community#131789] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### GitHub Feature Area

Security and Privacy

### Body

My account has been hacked two months ago, my email has been removed ( chadyachraf@outlook.com ) and password has changed..
I applied in the form two months ago right after I have been hacked, and a ticket has been created, but without any response ever since.
Here are my credentials : 
UserName: Chady00
Email: chadyachraf@outlook.com

Please help me. I can't access my account anymore.
I am really disappointed at how Github support is not able to assist me although this is an urgent concern.

**Comment by Olayiwolaaa** [comment]
I don't think that account is still active. 
Might have even been changed.

Do you still have access to the email?

**Reply by ghost** [reply]
The account might have been temporarily suspended by github. My email was altered, I knew it because I received this :

**_An email address (legion_scarlett877@fthcapital.com) was added to your account._**
then I received this: 
**_An email address (chadyachraf@outlook.com) was removed from your account._**


I still have access to the email, what should I do ?? I want my account back it has so many important projects
I have all evidence required to prove that this is my account

**Comment by queenofcorgis** [comment, marked as answer]
Hi @Chady001, 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding!

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!

Please note for future discussions:  You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.
